src/psyclone/psyir/nodes/reference.py

```diff
@@ -230,15 +180,12 @@ def create(name, children):
             are not of the expected type.
 
         '''
-        if not isinstance(name, str):
-            raise GenerationError(
-                "name argument in create method of Array class "
-                "should be a string but found '{0}'."
-                "".format(type(name).__name__))
-        if not name:
+        from psyclone.psyGen import GenerationError
```

Comment:
pylint complains that GenerationError has already been imported.

Reply:
Removed.